### DISCUSSION TITLE
FlatList items are not loading until CountDown Circle Timer onComplete

### DISCUSSION THREAD

**Discussion by dorapax-lab** [vydimitrov/react-countdown-circle-timer#147] [Q&A] [answered]
Hi,

In my app each item has CountDown Circle Timer. The list render 3 items per batch. When I scroll down list renders another 3 items and so on. But the problem is since item has CountDown Timer running,  FlatList items are not loading when scroll down until CountDownTimer onComplete, FlatList only renders renderListFooterComponent (not other items). I have to wait until CountDown Timer onComplete for render other items.

Is this because count down timer always rendering and it does not allow FlatList to render other items ? 
Also when I paused the timer i.e. `isPlaying={false}`, list renders items without any problem.

This is my FlatList config
```
         <FlatList
            data={items}
            renderItem={renderItem}
            keyExtractor={(item) => item._id}
            initialNumToRender={3}
            maxToRenderPerBatch={3}
            windowSize={3}
            onViewableItemsChanged={onViewRef.current}
            viewabilityConfig={viewConfigRef.current}
            onEndReached={() => handleOnEndReached()}
            ListFooterComponent={() => renderListFooterComponent()}
            ListHeaderComponent={() => renderListHeaderComponent()}
          />
```

**Comment by vydimitrov** [comment, marked as answer]
Hey, it seems the issue is very much tight to your implmentaion. If you create a demo reproducing the issue I can take a look on it.

**Reply by dorapax-lab** [reply]
Hi, Thanks for the reply. I'll try to make a demo simple as possible.
